sw/device/lib/dif/dif_csrng_shared.c

```diff
@@ -34,7 +36,12 @@ dif_result_t csrng_send_app_cmd(mmio_region_t base_addr, ptrdiff_t offset,
   // Build and write application command header.
   uint32_t reg = bitfield_field32_write(0, kAppCmdFieldCmdId, cmd.id);
   reg = bitfield_field32_write(reg, kAppCmdFieldCmdLen, cmd_len);
-  reg = bitfield_bit32_write(reg, kAppCmdBitFlag0, cmd.entropy_src_enable);
+  reg = bitfield_field32_write(
+      reg, kAppCmdFieldFlag0,
+      // When entropy src is enabled, we just need to program "not true".
+      (cmd.entropy_src_enable == kDifCsrngEntropySrcToggleDisable)
+          ? kMultiBitBool4True
+          : 0);
```

Comment:
I suspect this should be:
```suggestion
          : kMultiBitBool4False);
```

Reply:
yeah. Except if you do that, it triggers a lot of downstream changes in the unittest.
Functionally, writing a 0 here and writing `False` actually behaves the same.  Since the logic strictly looks for the "true" to use a fixed seed.